app/views/idv/by_mail/enter_code_rate_limited/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= render(
+      'idv/shared/error',
+      title: t('idv.gpo.title'),
+      heading: t('idv.failure.gpo.rate_limited.heading'),
+      options: [
+        {
+          url: MarketingSite.contact_url,
+          text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
+          new_tab: true,
+        },
+      ].select(&:present?),
```

Comment:
There are no conditional entries in the `options` so I think we can drop the `select` here

```suggestion
      ],
```